system/View/View.php

```diff
@@ -120,9 +120,9 @@ class View implements RendererInterface
 	 * The name of the current section being rendered,
 	 * if any.
 	 *
-	 * @var string|null
+	 * @var array<string>
 	 */
-	protected $currentSection;
+	protected $sectionStack = [];
```

Comment:
We cannot just replace this as this may be used in others' code. Maybe a deprecation would be okay.